clients/csharp-dotnet-core/voice-assistant-test/tool/DialogReport.cs

```diff
@@ -1,61 +1,61 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-namespace VoiceAssistantTest
```

Comment:
I'm confused about what has changed here and why it's showing big chunks have changed?

Reply:
This was a clrf line change that i accidentally clicked yet - nothing changed in this file so please ignore this